stable/airflow/UPGRADE.md

```diff
@@ -4,6 +4,7 @@
 
```

Comment:
Please add a new section here called: `v7.4.X` → `v7.5.0`

And write your new value under there.

Reply:
Don't forget to link this new anchor in the `README.md`

Reply:
fixed in [this commit](https://github.com/helm/charts/pull/23441/commits/a07c27a72b7226d5801e3de49fc80d16551a2bff)